com.stansassets.scene-management/Editor/Visualizer/StateStackVisualizer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine.UIElements;
+
+namespace StansAssets.SceneManagement
+{
+    public static class StateStackVisualizer
+    {
+
+        static readonly List<IStateStackVisualizerController> StackMap = new List<IStateStackVisualizerController>();
```

Comment:
I assume that if I will register the same stack multiple times - the system will tell me nothing about that and I will have N of the same visualization without any exception/error/warning.

```suggestion
        static readonly Dictionary<int, IStateStackVisualizerController> StackMap = new Dictionary<int, IStateStackVisualizerController>();
```
Where key (Int) is some unique Id of a Stack. We can't use just Stack as a key because of T generic parameter.